posthog/api/test/test_stickiness.py

```diff
@@ -165,6 +181,41 @@ def test_stickiness(self):
             self.assertEqual(response[0]["labels"][6], "7 days")
             self.assertEqual(response[0]["data"][6], 0)
 
+        @override_settings(PERSON_ON_EVENTS_V2_OVERRIDE=True)
+        @snapshot_clickhouse_queries
+        def test_stickiness_with_person_on_events_v2(self):
+            # KLUDGE: We need to do this to ensure create_person_id_override_by_distinct_id
+            # works correctly. Worth considering other approaches as we generally avoid
+            # truncating tables in tests.
+            sync_execute("TRUNCATE TABLE sharded_events")
+            self._create_multiple_people()
+            create_person_id_override_by_distinct_id("person1", "person2", self.team.pk)
```

Comment:
the approach we've used in other tests in the past is to use separate teams

Reply:
we still do this, but we separate teams per suite. unfortunately I need this specific test to have a clean slate. But yeah, wrote the comment here as one approach would be to create a new team